bloopgun/src/main/scala/bloop/bloopgun/Bloopgun.scala

```diff
@@ -587,14 +581,19 @@ class BloopgunCli(
     }
   }
 
+  private def helpRequested(cmd: String, cmdArgs: Array[String]): Boolean = {
+    def isHelp(str: String): Boolean = str == "-h" || str == "--help"
```

Comment:
Does this handle `bloop help`? I think it's nice to be flexible and allow `bloop -h`, `bloop -help`, `bloop help` and `bloop --help`

Reply:
Good point, I added `-help` and `help` as markers 👍